docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
+the other is the web-app that display visualized training data.
+
+The web-app reads the extract data and draw scalars graph or histograms graphs.
```

Comment:
I don't follow `reads the extract data and draw...`. I think once you have split the server side from the client side, you will be able to split the function as `reads the logs from disk` and `visualizes... to a web pages`

Reply:
Also, I think `web-app` is just web<space>app

Reply:
Yup, I combine the frontend and the backend server into one.

Reply:
^my typo. its web app. I had thrown in an XML tag |space|, but the markdown parser stripped it out

Reply:
Yeah i think the logical separation is: the `server` + `frontend` together being the frontend, and the rest being the sdk.